apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -84,6 +84,8 @@
 
     public static final OfficialComponent HYSTRIX =  new OfficialComponent(29, "Hystrix");
 
+    public static final OfficialComponent SOFARPC =  new OfficialComponent(30, "SOFARPC");
```

Comment:
I mean the value that it define in `component-libraries.yml`file isn't equals the value define in `ComponentsDefine.java` file

Reply:
@ascrutae They are all 30, only In test's component-libraries.yml ,I can not set it to 30, because jedis used It

Reply:
You can set the value to `32`, because the value define in `component-libraries.yml` is `32`. and more you also should change length of `components` filed to `33`

Reply:
@ascrutae I removed this settings

Reply:
all production code ,I need to set it as 30. last plugin is 29. I removed test settings, I did not use it in test project

Reply:
The component id must equals the `ComponentsDefine.java` defined's, or the component in topology will incorrect

Reply:
@ascrutae Ok,I will set it 32, SQLite(31) is wrong before? It has no defination in 
ComponentsDefine